### ISSUE TITLE
Question on custom kernel

### ISSUE THREAD

**Issue by kykean** [friendlyarm/NanoHatOLED#1]
Hi there,

Does this software require custom kernel to display on Nanopi NEO ? Or using an official armbian image will do the job ?

**Comment by wuweidong0107** [comment]
1. refer to http://wiki.friendlyarm.com/wiki/index.php/NanoPi_NEO, download nanopi-neo_ubuntu-oled_4.x.y_YYYYMMDD.img.zip.
2. Armbian image is not an FriendlyElec officail ROM. So We can't make sure if it supports OLED.

**Comment by kykean** [comment]
@wuweidong0107  Thanks. I will use that image and compile the tun.ko module for openVPN since it is missing from the image.

**Comment by overrider** [comment]
@kykean Did you compile tun.ko and can share it? I need it also..

**Comment by kykean** [comment]
@overrider Yes i did compiled the tun.ko, and edited the Python to show VPN status with button to restart it too. Let me fork this and share with you tomorrow.
UPDATE: i've uploaded the tun.ko on the forked repo https://github.com/kykean/NanoHatOLED. You can find plenty tutorials on loading .ko files, let me know if you need help.

**Comment by overrider** [comment]
@kykean Perfect, worked like a charme. Thanks

**Comment by overrider** [comment]
@kykean Could we recompile this module to work with the latest 4.14.0 kernel? I will try to contact the author for this friendly arm nano pi debian build, maybe we can just leave the tun module in.
